client/signup/jetpack-connect/install-step.jsx

```diff
@@ -7,20 +7,42 @@ import React from 'react';
  * Internal dependencies
  */
 import Card from 'components/card';
+import { preventWidows } from 'lib/formatting';
 
 export default React.createClass( {
 	displayName: 'JetpackInstallStep',
 
+	renderText() {
+		const { action } = this.props;
+		let { text } = this.props;
+
+		if ( ! action ) {
+			return (
+				<span>
+					{ preventWidows( text ) }
+				</span>
+			);
+		}
+
+		return (
+			<span>
+				{ text }
+				<span> </span>
```

Comment:
Correct, it is for forcing a whitespace. 

@rickybanister The action is the link in the first step of the screenshot above. So, the space is to separate the text from the link.